debugger/src/main/kotlin/org/rust/debugger/UpdatePrettyPrintersComponent.kt

```diff
@@ -25,7 +25,7 @@ class UpdatePrettyPrintersComponent : ApplicationComponent, Disposable {
         val application = ApplicationManager.getApplication()
         if (application.isUnitTestMode) return
         application.executeOnPooledThread {
-            checkPrettyPrinters()
+            checkPrettyPrinters(true)
```

Comment:
I personally find it more readable, (especially on GitHub, where there are no parameter hints), when `true` and `false` arguments use named parameter syntax: `checkPretyPrinters(clearOld = true)` ;)